<ui version="4.0" >
 <class>QVolume3DViewTestingExtensionBase</class>
 <widget class="QWidget" name="QVolume3DViewTestingExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>631</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_axialOrientationButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/axial.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_sagitalOrientationButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/sagital.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_coronalOrientationButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/coronal.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_zoomToolButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/zoom.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_rotate3DToolButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/rotate3d.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_panToolButton" >
       <property name="text" >
        <string>Axial</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../main/main.qrc" >:/images/move.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="udg::QTransferFunctionEditorByGradient" native="1" name="m_transferFunctionEditorByGradient" />
     <widget class="udg::Q3DViewer" native="1" name="m_3DView" />
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QTransferFunctionEditorByGradient</class>
   <extends>QWidget</extends>
   <header>qtransferfunctioneditorbygradient.h</header>
  </customwidget>
  <customwidget>
   <class>udg::Q3DViewer</class>
   <extends>QWidget</extends>
   <header>q3dviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../main/main.qrc" />
 </resources>
 <connections/>
</ui>
